tutorials/README.md

```diff
@@ -1,6 +1,6 @@
 # NoisePy CLI Tutorial
 
-NoisePy provides a CLI (command line interface) for running processing jobs. This tutorial works through the same tasks as the [get_started.ipynb](../get_started.ipynb) Jupyter Notebook.
+NoisePy provides a CLI (command line interface) for running processing jobs. This tutorial works through the same tasks as the **Noisepy Colab Tutorial** from the `tutorials/get_started.ipynb` Jupyter Notebook.
```

Comment:
This will make sense for people browsing the repo, but maybe a bit less to people looking at the rendered JupyterBook? 
@mdenolle WDYT?

Reply:
Sorry to be slow at responding . Ideally, we have one single place for tutorials that is the JupyterBook. So re-organizing the docs under a single framework should be the goal